packages/test/test-version-utils/src/compatConfig.ts

```diff
@@ -184,10 +184,18 @@ const genBackCompatConfig = (compatVersion: number): CompatConfig[] => {
 const genFullBackCompatConfig = (): CompatConfig[] => {
 	const _configList: CompatConfig[] = [];
 
-	const [, semverInternal] = fromInternalScheme(codeVersion, true, true);
-
+	const [, semverInternal, prereleaseIndentifier] = fromInternalScheme(codeVersion, true, true);
 	assert(semverInternal !== undefined, "Unexpected pkg version");
-	const greatestMajor = semverInternal.major;
+
+	// Here we check if the release is an "rc" release. If so, we also need to account for internal releases when
+	// generating back compat configs. This is done by adding to the total number of majors we need to generate.
+	// getRequestedVersion() will handle converting the requested RC release into an internal release if necessary.
```

Comment:
I think this means "RC 1 is treated like major 9"?  Maybe include an example in the comment, this was a little hard for me to interpret.